wgengine/wglog/wglog.go

```diff
@@ -29,7 +30,11 @@ type Logger struct {
 func NewLogger(logf logger.Logf) *Logger {
 	ret := new(Logger)
 
-	wrapper := func(format string, args ...interface{}) {
+	unlimitLogf := func(format string, args ...interface{}) {
```

Comment:
I think this is identical to `logger.WithPrefix("<RATELIMITED>", logf)`.

At which point I think this commit mostly could be shrunk to:

```go
unlimited := logger.WithPrefix("<RATELIMITED>", fixup)
rlogf := logger.RateLimitedFn(unlimited, 5*time.Second, 5, 100)
```

Also, I'd like to suggest a `logger.Unlimited(logf)` function that's implemented using logger.WithPrefix, so that if we rework rate-limited opt out we can do it all in-package.

Reply:
withdrawing the wg ratelimit commit for now in favour of josh's version.